xbmc/music/MusicDatabase.cpp

```diff
@@ -5819,32 +6110,26 @@ bool CMusicDatabase::GetFilter(CDbUrl &musicUrl, Filter &filter, SortDescription
     option = options.find("artistid");
     if (option != options.end())
     {
-      int idArtist = static_cast<int>(option->second.asInteger());
+      int idArtist = (int)option->second.asInteger();
```

Comment:
talking about auto, it could be used here since we explicitly cast the value to int, it's clear that it's an int. Might be a bad example as auto is more typing but anyway, just thought I'd give an example